AmplifyPlugins/API/Podfile

```diff
@@ -1,6 +1,6 @@
 load '../../build-support/dependencies.rb'
 
-platform :ios, '11.0'
+platform :ios, '12.0'
 use_frameworks!
 
 include_build_tools!
```

Comment:
we'll also want to pick up AppSyncRealTimeClient 3.2.0 to get Starscream's privacy manifest https://github.com/aws-amplify/aws-appsync-realtime-client-ios/releases/tag/3.2.0